Makefile

```diff
@@ -35,6 +35,7 @@ S3_SECRET_ACCESS_KEY ?= ''
 TYPE_OF_MANIFEST ?= master
 
 CONTAINER_ENGINE ?= docker
+CONTAINER_PLATFORM ?= linux/amd64
```

Comment:
This should also added to the notes here: https://github.com/integr8ly/integreatly-operator/blob/master/openshift-ci/README.md